test/config_test.py

```diff
@@ -0,0 +1,53 @@
+from contextlib import contextmanager
+import pytest
+
+from oci.oci import config
```

Comment:
This import cause the tests to fail:
https://travis-ci.org/nirs/oci/jobs/509601292